src/Components/Articles/ArticlesContainer.jsx

```diff
@@ -0,0 +1,50 @@
+import ArticleCard from "./ArticleCard"
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+const ArticlesContainer = () => {
+    const [articles, setArticles] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+    const [errorMessage, setErrorMessage] = useState(undefined);
+
+useEffect(()=>{
+    setIsLoading(true)
+
+
+    axios.get(`https://rachels-nc-notes.herokuapp.com/api/articles`).then((response)=>{
+        setErrorMessage(undefined);
+        setIsLoading(false);
+        setArticles(response.data.articles)
+    }).catch((err)=>{
+        setIsLoading(false);
+        setErrorMessage(err.message)        
+    })
+}, [errorMessage]);
+
+
+
+return (     
+ <section className="articles-container">
+    {errorMessage ? <p>Error: {errorMessage}</p> : ""}
+    {isLoading ? <p>Loading...</p> : ""}
+
+    
```

Comment:
I like your thinking but I don't think we need to display the error from the backend here as it will say "internal server error". Maybe change the error message to something more helpful, like "Oops, this isn't working right now. Please try again later"